build.gradle

```diff
@@ -25,6 +25,9 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity6'
+
+    implementation 'org.apache.poi:poi:5.2.5'
```

Comment:
What is the purpose of this dependancy?

Reply:
fulfill the database from the XML file located in resources/peaksAndSummits/Wierzcholki-wzgorza-i-szczyty-gor-w-Polsce.xls
With first run, or if something going wrong always can use it to make "clear" database with Summits.